test/external/handle-helpers

```diff
@@ -6,15 +6,15 @@ class Helpers < Sensu::Handler
     puts event_summary
   end
 
-  def api_request(*)
+  def api_request(*_args)
     nil
   end
 
-  def stash_exists?(*)
+  def stash_exists?(*_args)
     nil
   end
 
-  def event_exists?(*)
+  def event_exists?(*_args)
```

Comment:
I get that same behaviour with and without the `_` so I don't think that is a problem.